frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -172,6 +181,46 @@ export const surveyLogic = kea<surveyLogicType>([
                 }
             },
         },
+        surveyRatingResults: {
+            loadSurveyRatingResults: async ({
+                questionIndex,
+                question,
+            }: {
+                questionIndex: number
```

Comment:
`questionIndex` is sufficient here, since you can get the `question` from `values.survey.questions[questionIndex]`